docs/DeveloperGuide.md

```diff
@@ -226,6 +243,52 @@ user in the `s-update` command
     * Cons: There are not many operations to do with `Drink`s. It is only used to represent a constant set of
      drink types.
      
+### \[Completed\] Listing Sales Data of the Drinks
+
+tCheck allows users to view a list of the recorded sales data of drinks. 
+The list of drinks shown is ordered in descending order (i.e. ranked from the most to least sales).
+The command to use this feature is: `s-list`.
+
+#### Implementation
+
+The completed mechanism to list the sales data in descending order is facilitated by the `UniqueSalesRecordList`.
+This is a list of `SalesRecordEntry`, containing the `numberSold` for a type of `Drink`. It implements the following
+operations:
+
+* `UniqueSalesRecordList#sort()`  —  Sorts the `SalesRecordEntry` in the list in descending order based on its
+ `numberSold`.
+* `UniqueSalesRecordList#asUnmodifiableObservableList()`  —  Returns the list as an unmodifiable form.
+ 
+These operations are exposed in the `SalesBook` as `SalesBook#sortRecord()` and `SalesBook#getSalesRecord()` 
+respectively. They are further exposed in the `Model` interface as `Model#sortSalesBook()` and 
+`Model#updateFilteredSalesRecordList(Predicate<SalesRecordEntry>)` respectively.
```

Comment:
Is `UniqueSalesRecordList#asUnmodifiableObservableList()` indeed exposed as `Model#updateFilteredSalesRecordList(Predicate<SalesRecordEntry>)`? I think that `Model#updateFilteredSalesRecordList(Predicate<SalesRecordEntry>)` is about filtering the list by predicate, while `UniqueSalesRecordList#asUnmodifiableObservableList()` means shows the list?

Reply:
It may not be that direct. I will remove this section as I cannot explain it that accurately.